core/src/main/java/nl/nn/adapterframework/stream/document/DocumentUtils.java

```diff
@@ -0,0 +1,132 @@
+/*
+   Copyright 2021 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.stream.document;
+
+import java.util.Map.Entry;
+
+import org.apache.commons.lang3.NotImplementedException;
+import org.xml.sax.SAXException;
+
+import jakarta.json.JsonArray;
+import jakarta.json.JsonNumber;
+import jakarta.json.JsonObject;
+import jakarta.json.JsonString;
+import jakarta.json.JsonValue;
+
+public class DocumentUtils {
+
+	public static String DEFAULT_ARRAY_ELEMENT_NAME = "item";
```

Comment:
hier nog appconstants property van maken of niet?

Reply:
als er om gevraagd wordt ... ?